Post/tests/test_url_posts.py

```diff
@@ -14,3 +14,7 @@ def test_post_detail_url_is_resolver(self):
     def test_createPost_url_is_resolved(self):
         url = reverse('createPost')
         assert resolve(url).func == CreateNewPost
+
+    def test_searchPost_url_is_resovled(self):
+        url = reverse('post_list_Search')
+        assert resolve(url).func == postListSearch
```

Comment:
can you please explain this test?

Reply:
@guy9050 yes.
This test checks that every URL is mapping to the view function we wanted to.